avro/src/test/java/com/fasterxml/jackson/dataformat/avro/schema/AvroSchemaGenerationTest.java

```diff
@@ -169,4 +172,38 @@ public void testSchemaForUntypedMap() throws Exception
             verifyException(e, "Maps with non-stringable keys are not supported (yet?)");
         }
     }
+
+    // Issue 388 Default value for enums with class
+    public void testClassEnumWithDefault() throws Exception
```

Comment:
I think this test better belongs to `EnumTest` class.

`EnumTest` did not exists at the time of this PR yet.

Reply:
Will move.